src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -143,6 +143,24 @@ struct YarpSensorBridge::Impl
         index = iter - vec.begin();
         return true;
     }
+    
+    /**
+     * Utility function to check for nan in vector
+     */
+    bool nanExistsInVec(const yarp::sig::Vector& vec, 
+                        std::string_view logPrefix,
+                        const std::string& sensorName)
+    {
+        for (auto& val : vec)
+        {
+            if (std::isnan(val))
+            {
+                std::cerr << logPrefix << " NAN values read from " << sensorName << " , use previous measurement" << std::endl;
+	        return true;
+	    }
```

Comment:
The indentation seems a little off

Reply:
parallax error :D

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/191/commits/bad48a1bb5338489ab4071153574cf98463494b4